src/inverseRobot.js

```diff
@@ -7,7 +7,28 @@
 */
 
 function inverseRobot(robot) {
```

Comment:
Hmm this code looks a bit complicated, you don't need two `for` loops to get object values and compare them, also to check if an object value appeared earlier and return null you should use the `hasOwnProperty()` method. Here is a simpler version:

```suggestion
function inverseRobot(robot) {
  const robotInverse = {};

  for (const key in robot) {
    if (robotInverse.hasOwnProperty(robot[key])) {
      return null;
    }

    robotInverse[robot[key]] = key;
  }

  return robotInverse;
}
```

Easier and cleaner to read 😃

Reply:
Thank you for suggestions :)